File: Data Structures & Algorithms/max-consecutive-ones/submission-0.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int ans = 0,cr = 0;
        for(int i : nums)
        {
            if(i == 1) cr+=1;
            else cr = 0;
            ans = Math.max(ans,cr);
        }
        return ans;
    }
}

File: Data Structures & Algorithms/products-of-array-discluding-self/submission-0.cpp
class Solution {
public:
    vector<int> productExceptSelf(vector<int>& nums) {
        vector<int>ans(nums.size(),0);
        int z = 0;
        int pr = 1;
        for(int i : nums)
        {
            if(i == 0) z+=1;
            else pr*=i;
            if(z >= 2) return ans;
        }
        for(int i = 0; i < nums.size(); i++)
        {
            if(z == 1 && nums[i] == 0) ans[i] = pr;
            else if(nums[i] != 0 && z == 1) ans[i] = 0;
            else ans[i] = pr/nums[i];
        }
        return ans;
    }
};
